GreenhouseNode/GreenhouseNode/tasks/LoRaWAN.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * LoRaWAN.h
+ *
+ * Created: 13/11/2021 12.10.30
+ *  Author: Michel Sofus Engelhardt Sommer, 273966
+ */ 
+
+/*
+ *SEP4_GROUP1
+ *Websocket Url: wss://iotnet.cibicom.dk/app?token=vnoUBQAAABFpb3RuZXQuY2liaWNvbS5ka4lPPjDJdv8czIiFOiS49tg=
+ *Http to check on loriot network server: https://iotnet.teracom.dk/apps/websocket.html?token=vnoUBQAAABFpb3RuZXQuY2liaWNvbS5ka4lPPjDJdv8czIiFOiS49tg=
+ *AppKey:5F83717BC67B4646E3F00DC5EC3417DC
+ *AppEUI:2BBE8F09765BBF4B
+ *EUI Dev#1:0004A30B00259D2C
+ *EUI Dev#2:0004A30B00251192
```

Comment:
I would suggest moving all of  this to the README file, so we don't have to keep it different places, along with the comments on line 26 and 29. It seems these IDs are commented in multiple different files, `LoRaWAN_handler.h` and `LoRaWAN_Task.c`.